Jakefile.js

```diff
@@ -139,11 +139,6 @@ compileOutputConfigFile('src/typescriptServices/tsconfig.json', [], [copyright],
     // Created by appending 'export = ts;' at the end of the standalone file to turn it into an external module
     var nodeDefinitionsFileContents = definitionFileContents + "\nexport = ts;";

```

Comment:
To support browser cases seamlessly, this should also probably append `"\nexport as namespace ts;"` now.

Reply:
fair point.

Reply:
I can get these done in a different PR.